tools/ds-identify

```diff
@@ -267,6 +288,10 @@ detect_virt() {
         if [ $r -eq 0 ] || { [ $r -ne 0 ] && [ "$out" = "none" ]; }; then
             virt="$out"
         fi
+    elif [ "$DI_UNAME_KERNEL_NAME" = "FreeBSD" ]; then
+        out=$(sysctl -qn hw.vm_guest 2>/dev/null) && {
```

Comment:
I kind of doubt that this matches up exactly with the output of [systemd-detect-virt](https://www.freedesktop.org/software/systemd/man/systemd-detect-virt.html)  and other parts of ds-identify inspect the value of DI_VIRT.

So we probably need to map its output somehow.

Reply:
for one, it's wrong.
it should be `kern.vm_guest` — which on my systems is `kvm`.
i blame this on a mixup with `hw.hv_vendor`, which on my system is `KVMKVMKVM`

aaaaaaaaaaand, i don't know how we can best test how well the outputs map.

Reply:
The answer is: The values do map:

https://github.com/freebsd/freebsd/blob/f9df8418bd7d40e3b251df41fb49ec93b998dcc3/sys/kern/subr_param.c#L149-L157

Reply:
addressing and mapping should now be fixed.

Reply:
wow. surprising that matches up so well.  Consistency is so rare.